code/src/components/AlbumName.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+const AlbumName=(props) => {
+    return(<a className='album-name' href={props.name.external_urls.spotify}>{props.name.name}</a>)
```

Comment:
We noticed that the link in play-icon was opening in a new browser tab, but this one not. Maybe it could be better to have a consistent way to do it with the links, either opening all in a new tab or all in the same tab? Just as a thought :)